guides/keras_core/migration_guide_tf_keras_3_0.py

```diff
@@ -0,0 +1,805 @@
+"""
+Title: Migration guide : Tensorflow to Keras 3.0
+Author: [Divyashree Sreepathihalli](https://github.com/divyashreepathihalli)
+Date created: 2023/10/23
+Last modified: 2023/10/30
+Description: Instructions on migrating your TensorFlow code to Keras 3.0.
+Accelerator: None
+"""
+
+"""
+This guide will help you migrate code from TensorFlow 2.x to keras 3.0. The overhead for
+the migration is minimal. But once you have migrated you can run Keras workflows on top
+of arbitrary frameworks — starting with TensorFlow, JAX, and PyTorch.
+
+Keras 3.0 is also a drop-in replacement for `tf.keras`, with near-full backwards
+compatibility with `tf.keras` code when using the TensorFlow backend. In the vast
+majority of cases you can just start importing it via `import keras` in place of `from
+tensorflow import keras` and your existing code will run with no issue — and generally
+with slightly improved performance, thanks to XLA compilation.
+
+Commonly encountered issues and frequently asked questions can be located in
+the following links.
+
+[Known
+Issues](https://keras.io/keras_core/announcement/#:~:text=Enjoy%20the%20library!-,Known%20issues,-Keras%20Core%20is)
+
+[FAQs](https://keras.io/keras_core/announcement/#:~:text=Frequently%20asked%20questions)
+"""
+
+"""
+## Setup
+
+First, lets install keras-nightly.
+
+We're going to be using the Tensorflow backend here -- but you can edit the string below
+to "tensorflow" or "torch" and hit "Restart runtime", once you have migrated your code
+and your code will run just the same!
+"""
+
+"""shell
+! pip install keras-nightly
+"""
+
+import os
+
+os.environ["KERAS_BACKEND"] = "tensorflow"
+
+import keras
+import tensorflow as tf
+import numpy as np
+
+"""
+## Switching Tensorflow code to Keras 3.0 - Tensorflow backend
+
+Follow these instructions to migrate your existing TensorFlow code to Keras 3.0 and run
+it with the TensorFlow backend:
+
+1.   Update imports : replace `from tensorflow import keras` to `import keras`
+2.   Update code : replace `tf.keras.*` to `keras.*`
+
+
+
+"""
+
+"""
+## Migration incompatabilities : `tf.keras` to `keras 3.0`
+Keras 3 is a significant milestone in the evolution of the Keras API. It features a
+number of cleanups and modernizations that have resulted in a few breaking changes
+compared to Keras 2. All APIs that were removed were dropped due to extremely low usage.
```

Comment:
It's hard to know what "extremely low usage" means. One user? One hundred users? Maybe just say, "low usage."